pyanaconda/modules/payloads/kickstart.py

```diff
@@ -302,6 +267,7 @@ class PayloadKickstartSpecification(KickstartSpecification):
     commands_data = {
         "ModuleData": COMMANDS.ModuleData,
         "RepoData": COMMANDS.RepoData,
+
```

Comment:
I think this might be a typo/leftover ? :)

Reply:
I think we can also remove these once it pykickstart it gets to RemovedCommand. 
I tried removing it and I got ~50 tests failures.